modular-docs-manual/master-docinfo.xml

```diff
@@ -0,0 +1,8 @@
+<productname>FCC</productname>
```

Comment:
Why is this file included?

Reply:
This file and its components are required in order to build the ccutil version of the guide. It was half way set up to be able to do so, so I just fixed the rest to enable it, for publishing on Pantheon if we need to. We can make the `productname` whatever, but needs to be something there in order to build. So I chose FCC. Since it would only be used internally anyway, maybe doesn't matter.

Reply:
This doc cannot be published through Pantheon because it's on GH, not internal GL. If anyone feels the need to build it through Pantheon, they will still need to set it up on GL where all RH-specific prereqs can be satisfied.

(Again, I believe we should keep this project as much org-agnostic as possible. Otherwise we're sending a msg of "This is a RH thing, and it's not suitable for wider adoption." RH has many upstream (docs) projects that get periodically downstreamed for publishing internally. None of those projects put downstream docinfo files into their upstream community repos.)

Reply:
Done. Removed.